code/datums/elements/falling_hazard.dm

```diff
@@ -0,0 +1,80 @@
+/// An element that will make a target thing do damage to any mob that it falls on from a z-level above
+/datum/element/falling_hazard
+	element_flags = ELEMENT_BESPOKE
+	id_arg_index = 2
```

Comment:
unless i'm crazy this means you're indexing by the damage which means every element with the same damage is going to have the same values for everything else based on the last element that gets created

Reply:
arguable you could make a type index that would sort of make sense if you add this element to a lot of things

Reply:
nevermind i'm wrong this is actually where it "Starts" hashing for uniqeness